blog/2019-10-02-MigratingOptaWebEmployeeRosteringToSpringBoot.adoc

```diff
@@ -0,0 +1,204 @@
+= What's New in OptaWeb Employee Rostering? (part 2) - Making the Backend Cloud-Ready
+:page-interpolate: true
+:awestruct-author: jucui
+:awestruct-layout: blogPostBase
+:awestruct-tags: [employee rostering, coding, algorithm, production]
+
+Recently, the OptaPlanner team has been shifting towards cloud-native applications.
+As a part of this effort, we migrated the OptaWeb Employee Rostering Backend Application from WildFly to Spring Boot.
+
+OptaWeb Employee Rostering is a web application that optimizes shift scheduling for maximum efficiency and fairness for employees.
+In this post we’ll give an overview of the new Spring backend application, and the key features that were added on top of the existing backend.
```

Comment:
we’ll -> we'll
The ' is different. Use this type of '. Word tend to throw in the other kind.

Reply:
Strange, I have the correct "we'll" in my IDE, but when I push GitHub shows the former symbol. Is there an IDE setting I'm missing?